systemtest/src/main/java/io/strimzi/systemtest/utils/TestWatcher.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.utils;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.extension.AfterTestExecutionCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.io.File;
+import java.lang.reflect.Method;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+import static io.strimzi.systemtest.AbstractST.TEST_LOG_DIR;
+import static io.strimzi.test.BaseITST.CLIENT;
+import static io.strimzi.test.BaseITST.KUBE_CLIENT;
+
+public class TestWatcher implements AfterTestExecutionCallback {
```

Comment:
We already have `TestWatcher` class in JUnit library so maybe to prevent conflicts we should rename this class?

Reply:
Make sense